examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2466,6 +2503,124 @@ impl FrontendNode {
         expect_matches!(result, Err(_));
         Ok(())
     }
+
+    fn test_grpc_server_pseudo_node_privilege(&mut self) -> TestResult {
+        // gRPC server pseudo node can be created even with Top confidentiality.
+        let top_label = oak_abi::label::confidentiality_label(oak_abi::label::top());
+        let config = oak::node_config::grpc_server(ADDITIONAL_TEST_SERVER_ADDR);
+        let (wh, rh) = oak::channel_create("Test", &top_label).expect("could not create channel");
+        expect_eq!(
+            Ok(()),
+            oak::node_create("grpc_server", &config, &top_label, rh)
+        );
+        expect_eq!(Ok(()), oak::channel_close(rh.handle));
+        expect_eq!(Ok(()), oak::channel_close(wh.handle));
+        Ok(())
+    }
+
+    fn test_http_server_pseudo_node_privilege(&mut self) -> TestResult {
+        // HTTP server pseudo node can be created even with Top confidentiality.
+        let top_label = oak_abi::label::confidentiality_label(oak_abi::label::top());
+        let config = oak::node_config::http_server(ADDITIONAL_TEST_SERVER_ADDR);
```

Comment:
Should this be changed to `ADDITIONAL_TEST_SERVER_ADDR_2`? If not, perhaps we need a better name for `ADDITIONAL_TEST_SERVER_ADDR_2`.

Reply:
Changed it to ADDITIONAL_TEST_SERVER_ADDR_2.